_includes/markdown/Markup.md

```diff
@@ -285,10 +285,69 @@ Combining automated and manual testing practices allows 10up to maintain a high
 
-At 10up, we employ progressive enhancement to ensure that the sites we build for our clients are accessible to as many users as possible. For example, browser support for SVG has not yet reached 100%. When using SVG you should always provide a fallback such as a PNG image for browsers that do not support vector graphics.
+At 10up, we employ progressive enhancement to ensure that the sites we build for our clients are accessible to as many users as possible.
 
 ### Polyfills
 When writing markup that does not have wide browser support, using polyfills can help bring that functionality to those older browsers. Providing support for older browsers is incredibly important to the business objectives of our clients. In an effort to prevent code bloat, we only provide polyfills for features that are functionally critical to a site.
 
 ### Feature Detection
 At 10up, the concept of feature detection is used to test browser support for new features that do not yet have full support across the board. The concept of feature detection is to test if a feature is supported by the browser and if not supported, conditionality run code to provide a similar experience with browsers that do support the feature. While popular [feature detection libraries](https://modernizr.com/) exist, there are [feature detection techniques](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection#JavaScript) for JavaScript and [@supports](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports) at-rule for CSS that can be utilized.
+
+<h2 id="svg" class="anchor-heading">SVG {% include Util/top %}</h2>
+<abbr title="Scaleable Vector Graphics">SVG</abbr> has become a prevalent means for displaying rich vector graphics. Here are a few known benefits of SVG:
+
+* __Scalability__ - They look great on retina displays and at any size, i.e. they're resolution independent.
+* __File Size__ - Small file size and compresses well.
+* __Styling__ - Manipulate fill, stroke, and even animate.
+
+Be mindful that SVGs have potential drawbacks as well:
+
+* __Security__ - Blindly including an SVG document in your page has the potential to introduce a security vulnerability. This is why WordPress does not allow uploading of SVGs by default. Read: [SVG uploads in WordPress (the Inconvenient Truth)](https://bjornjohansen.no/svg-in-wordpress) for more information.
+* __Anti-patterns__ - Some uninformed users will embed a raster (`.jpg`, `.png`, `.gif`) file within an SVG document thinking this will give them the same affordances of SVGs (crispness and ability to style). Ultimately, it is still just a raster file within an SVG document and will be rendered as such.
```

Comment:
This is somewhat what I had in mind, but it sounds kind of accusatory. I was thinking of a more "positive" statement of best practice about when to use SVGs, when raster formats are more appropriate, and what happens when you try to combine them. Something more along the lines of:

* SVG is a vector format, so it excels at images with well-defined lines and simple color palettes that can be defined algorithmically. Examples: logos and logotype, iconography, illustrations, etc.
* SVG is NOT as good at rendering more photographic images—for images with complex visual data, raster formats such as PNG, JPEG, or GIF are probably still better options.
* Raster images *cannot* be converted to vector, so you can't convert an existing PNG to SVG, for example. This will just result in the raster image being embedded within an SVG document and will not provide the full benefits of using SVG.
* For further reading on vector vs. raster formats, and when to use each: https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Adding_vector_graphics_to_the_Web

This is just off the top of my head, so feel free to take/leave/modify any of it.

Reply:
@dkoo I really appreciate the bullet points here. I added them in. Please let me know if it makes sense where I added it? 

And, any other suggestions you might have. 👍